log/logrus/logrus_logger.go

```diff
@@ -0,0 +1,31 @@
+package log
```

Comment:
Now that the code is moved into the log/logrus package, the package name here should also be logrus.

Reply:
Please add at least a sentence of package documentation.

Reply:
Can you please clarify on what you mean by package documentation? I looked at other packages like endpoint.go and metrics.go and didn't see anything other that function and type docs.